### PR TITLE
Get the ibm-tutorial example working

### PR DESCRIPTION
**Pull request by bradbeck**

### CODE REVIEW

File: examples/ibm-tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# IBM Tutorial Demo
+
+## Demo
+
+```bash
+# Only if a cluster is needed.
+make setup-minikube
+
+# Setup tekton w/ chains
+make setup-tekton-chains tekton-generate-keys setup-kyverno
```
Comment (rgreinho) [inline examples/ibm-tutorial/README.md:10]: It does not hurt to install it, but kyverno is not needed for this example.

**Review by rgreinho — CHANGES_REQUESTED** [review]
The installation worked like a charm, but I could not get the port forwarding to work.

File: examples/ibm-tutorial/README.md (hunk 1 above)
Comment (bradbeck) [reply examples/ibm-tutorial/README.md:10]: It seems like we typically want the admission controller in place for the pipeline runs. In this case it caught an outdated version of kaniko.

File: examples/ibm-tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## In a separate terminal, port forward the registry: defaults to localhost:8888
+#  ./platform/05-minikube-registry-proxy.sh
```
[the remark this answers is not in the document]
Comment (bradbeck) [reply examples/ibm-tutorial/README.md:13]: Is it minikube version related? I'm using the following version:
```
minikube version: v1.24.0
commit: 76b94fb3c4e8ac5062daf70d60cf03ddcc0a741b
```

[the remark this answers is not in the document]
Comment (rgreinho) [reply examples/ibm-tutorial/README.md:13]: Same one: 

```
╭ ~
╰❯ minikube version
minikube version: v1.24.0
commit: 76b94fb3c4e8ac5062daf70d60cf03ddcc0a741b
```

[the remark this answers is not in the document]
Comment (bradbeck) [reply examples/ibm-tutorial/README.md:13]: I get:
```
$ ./platform/05-minikube-registry-proxy.sh
Port forwarding the minikube registry: REGISTRY=localhost:8888
e.g.: curl http://localhost:8888/v2/_catalog
Forwarding from 0.0.0.0:8888 -> 5000

[the remark this answers is not in the document]
Comment (rgreinho) [reply examples/ibm-tutorial/README.md:13]: Now I got it to work! 👍

**Review by rgreinho — APPROVED** [review]

File: examples/ibm-tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Or re-run the last one.
+# tkn pipeline start build-and-deploy-pipeline -L
+
+# Export name of the docker image from the pipelinerun as DOCKER_IMG:
+IMAGE_URL=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageUrl") | .value')
+export IMAGE_URL=localhost:8888${IMAGE_URL#"registry.kube-system.svc.cluster.local"}
+IMAGE_TAG=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageTag") | .value')
+export IMAGE_TAG
+DOCKER_IMG="${IMAGE_URL}:${IMAGE_TAG}"
+export DOCKER_IMG
```
Comment (rgreinho) [inline examples/ibm-tutorial/README.md:26]: Nothing wrong nor blocking here, but a remark to squeeze the lines a little:

```suggestion
IMAGE_URL=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageUrl") | .value')
export IMAGE_URL=localhost:8888${IMAGE_URL#"registry.kube-system.svc.cluster.local"}
export IMAGE_TAG=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageTag") | .value')
export DOCKER_IMG="${IMAGE_URL}:${IMAGE_TAG}"
export COSIGN_KEY="k8s://tekton-chains/signing-secrets"
```

**Review by rgreinho — APPROVED** [review]
Just one more small remark, but everything works fine for me now. Good job 👍

**Review by mlieberman85 — APPROVED** [review]

**Pull request merged**